### PR TITLE
Created to-radian function for java.

### PR DESCRIPTION
**Pull request by manish-singh-bisht**
Created to radian function for java.
Solved issue #921.

### CODE REVIEW

**Comment by manish-singh-bisht** [comment]
Hi completed the task ,please review it.

**Comment by manish-singh-bisht** [comment]
Hey if there's any issue, I will resolve it.  Thank you.

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@manish-singh-bisht, I completed the first review and left some comments. Note you still need to sign the [CLA](https://cla-assistant.io/Codecademy/docs?pullRequest=1048).

**Comment by manish-singh-bisht** [comment]
Hey resolved all the issues,please review it,

File: content/java/concepts/math-methods/terms/to radian/to-radian.md
Comment (SSwiniarski) [inline content/java/concepts/math-methods/terms/to radian/to-radian.md:15]:
```suggestion
The **`Math.toRadians`** method returns the measurement of an angle in degrees to an equivalent angle in radians.
```

File: content/java/concepts/math-methods/terms/to radian/to-radian.md
Comment (SSwiniarski) [inline content/java/concepts/math-methods/terms/to radian/to-radian.md:24]:
```suggestion
The conversion from degrees to radians is generally inexact.
```

File: content/java/concepts/math-methods/terms/to radian/to-radian.md
Comment (SSwiniarski) [inline content/java/concepts/math-methods/terms/to radian/to-radian.md:34]:
```suggestion
    double deg1 = 180.0;
```

File: content/java/concepts/math-methods/terms/to radian/to-radian.md
Comment (SSwiniarski) [inline content/java/concepts/math-methods/terms/to radian/to-radian.md:35]:
```suggestion
    double deg2= 45.0;
```

**Review by SSwiniarski — COMMENTED** [review]
@manish-singh-bisht, I had a few more comments.

**Comment by manish-singh-bisht** [comment]
Hey solved the issues, please review it.

**Comment by manish-singh-bisht** [comment]
Hey solved all mentioned changes , please review it.

**Review by SSwiniarski — APPROVED** [review]
@manish-singh-bisht, Looks good for the second review.

**Comment by yangc95** [comment]
> Hey will it get merged?

Hi @manish-singh-bisht, the review process requires two maintainers to approve prior to merging. Currently, there is one approval, so half-way there!

File: content/java/concepts/math-methods/terms/to radian/to-radian.md
Comment (HishamT) [inline content/java/concepts/math-methods/terms/to radian/to-radian.md:15]: ~Math.toRadians~ Math.toRadians()

File: content/java/concepts/math-methods/terms/to radian/to-radian.md
Comment (HishamT) [inline content/java/concepts/math-methods/terms/to radian/to-radian.md:28]: ~The following example uses `Math.toRadians()` to return radian measurement of 180 and 45 degrees:~ 

Consider the following sentence instead:

The following example uses `Math.toRadians()` to convert 180 degrees and 45 degrees to radians:

**Review by HishamT — CHANGES_REQUESTED** [review]
Finished second review. Left some suggestions.

**Comment by manish-singh-bisht** [comment]
Hey @HishamT ,please review it.

**Review by HishamT — APPROVED** [review]
LGTM

**Review by yangc95 — COMMENTED** [review]
@manish-singh-bisht this almost looks ready for merge!

I just had one suggestion and also a request to change the directory name from `to radian` to `to-radian`

**Comment by manish-singh-bisht** [comment]
Hey @yangc95  please review it.Thank you.

**Review by yangc95 — APPROVED** [review]

**Comment by yangc95** [comment]
@manish-singh-bisht merged! I made a small edit here [03be8f0](https://github.com/Codecademy/docs/pull/1048/commits/03be8f0b19093854b9966506c191fc9a1c887550) prior to merge

link: https://www.codecademy.com/resources/docs/java/math-methods/to-radian

cc: @HishamT @SSwiniarski

**Comment by manish-singh-bisht** [comment]
@yangc95 Thank you for the advice of creating a new branch, I will definitely follow this practice. I will choose to remain anonymous. Thank you.

**Pull request merged**
